status/code.go

```diff
@@ -0,0 +1,91 @@
+package status
```

Comment:
I think the useage will be more clear if this wasn't it's own package.

```go
status.CodeSuccess
// Vs
epplib.StatusCodeSuccess
```

WDYT?

Reply:
It's probably more clear for all of the code in this repository if it's in the root but it wouldn't really be a nice structure IMO.
Would it be sufficient to rename the package to eppstatus or something instead?

Reply:
I think it would be more clear for users that depend on this repository if they would use `epplib.StatusCodeSuccess` just like they use `http.StatusAccepted`. Speaking of, perhaps  it can be called `epplib.StatusSuccess` or `status.Success`?

I'm fine ether way though.